package.json

```diff
@@ -24,19 +24,20 @@
   "scripts": {
     "build": "vite build",
     "test:code": "jest --collect-coverage",
-    "test:types": "jest -c jest.config.tsd.js --collect-coverage",
+    "test:types": "jest -c jest.config.tsd.js",
```

Comment:
`--collect-coverage` has no effect, because no code is executed during type testing.